src/frameworks/next/index.ts

```diff
@@ -229,7 +260,14 @@ export async function build(dir: string): Promise<BuildResult> {
     console.log("");
   }
 
-  return { wantsBackend, headers, redirects, rewrites, trailingSlash };
+  return {
+    wantsBackend,
+    headers,
+    redirects,
+    rewrites,
+    trailingSlash,
+    ...(!nextjsI18n?.domains ? { i18n: nextjsI18n } : {}),
```

Comment:
Change to just `i18n,` and above let's set `const i18n = nextjsI18n && !nextjsI18n.domains ? nextjsI18n : undefined;` that way we don't need a ternary in a splat